File: v2.x/Examples/SciChartDemo/ShareChartExample/ShareChartExample/ViewController.m
//
//  ViewController.m
//  ShareChartExample
//
//  Created by Admin on 5/26/16.
//  Copyright © 2016 Admin. All rights reserved.
//

#import "ViewController.h"
#import "ChartView.h"

@interface ViewController () <UIAlertViewDelegate>

@end

@implementation ViewController {

}

- (void)viewDidLoad {
    [super viewDidLoad];
    ChartView *chartView = [[ChartView alloc] initWithFrame:self.view.bounds];
    chartView.autoresizingMask = UIViewAutoresizingFlexibleWidth|UIViewAutoresizingFlexibleHeight;
    chartView.translatesAutoresizingMaskIntoConstraints = YES;
    [self.view addSubview:chartView];
}

@end

File: v2.x/Tutorials/Swift3/Tutorial02/HavingFunWithSciChart/ViewController.swift
//
//  ViewController.swift
//  HavingFunWithSciChart
//
//  Created by Yaroslav Pelyukh on 31/01/2017.
//  Copyright © 2017 SciChart Ltd. All rights reserved.
//

import UIKit
import SciChart

class ViewController: UIViewController {
    
    var sciChartSurface: SCIChartSurface?
    
    override func viewDidLoad() {
        super.viewDidLoad()
        
        // Create a SCIChartSurface. This is a UIView so can be added directly to the UI
        sciChartSurface = SCIChartSurface(frame: self.view.bounds)
        sciChartSurface?.translatesAutoresizingMaskIntoConstraints = true
        
        // Add the SCIChartSurface as a subview
        self.view.addSubview(sciChartSurface!)
        
        // Create an XAxis and YAxis. This step is mandatory before creating series
        sciChartSurface?.xAxes.add(SCINumericAxis())
        sciChartSurface?.yAxes.add(SCINumericAxis())

        // That's it! The SCIChartSurface will now display on the screen with default axis ranges
    }
}
